### PR TITLE
Add CN README in the Homepage

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: README_cn.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  - 时空动作检测：SlowOnly + Fast-RCNN, SlowFast + Fast-RCNN
+
+  具体可参考 [ModelZoo](#模型库)
+
+- **详尽的单元测试和文档**
+  MMAction2 提供了详尽的说明文档，API 接口说明，全面的单元测试，以供社区参考
+
+## 更新记录
+
+v0.10.0 版本已于 2021 年 1 月 5 日发布，可通过查阅 [changelog.md](docs/changelog.md) 了解更多细节以及发布历史
```
Comment (innerlee) [inline README_cn.md:52]: 0.11 has been released

File: README_cn.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+## 数据集准备
+
+请参考 [data_preparation.md](/docs/data_preparation.md) 了解数据集准备概况。所有支持的数据集都列于 [supported_datasets.md](docs/supported_datasets.md) 中
+
+## 教程
+
+请参考 [getting_started.md](/docs/getting_started.md) 了解 MMAction2 的基本使用。MMAction2也提供了其他更详细的教程：
+[learn about configs](docs/tutorials/1_config.md), [finetuning models](docs/tutorials/2_finetune.md), [adding new dataset](docs/tutorials/3_new_dataset.md),
+[designing data pipeline](docs/tutorials/4_data_pipeline.md), [adding new modules](docs/tutorials/5_new_modules.md), [exporting model to onnx](docs/tutorials/6_export_model.md) 和 [customizing runtime settings](docs/tutorials/7_customize_runtime.md).
```
Comment (dreamerlin) [inline README_cn.md:174]: Should these tutorial name be translated to Chinese ?

File: README_cn.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+- [x] [Kinetics-[400/600/700]](/tools/data/kinetics/README.md) \[ [Homepage](https://deepmind.com/research/open-source/kinetics) \] (CVPR'2017)
+- [x] [Something-Something V1](/tools/data/sthv1/README.md) \[ [Homepage](https://20bn.com/datasets/something-something/v1) \] (ICCV'2017)
+- [x] [Something-Something V2](/tools/data/sthv2/README.md) \[ [Homepage](https://20bn.com/datasets/something-something) \] (ICCV'2017)
+- [x] [Moments in Time](/tools/data/mit/README.md) \[ [Homepage](http://moments.csail.mit.edu/) \] (TPAMI'2019)
+- [x] [Multi-Moments in Time](/tools/data/mmit/README.md) \[ [Homepage](http://moments.csail.mit.edu/challenge_iccv_2019.html) \] (ArXiv'2019)
+- [x] [HVU](/tools/data/hvu/README.md) \[ [Homepage](https://github.com/holistic-video-understanding/HVU-Dataset) \] (ECCV'2020)
+- [x] [Jester](/tools/data/jester/README.md) \[ [Homepage](https://20bn.com/datasets/jester/v1) \] (ICCV'2019)
+- [x] [GYM](/tools/data/gym/README.md) \[ [Homepage](https://sdolivia.github.io/FineGym/) \] (CVPR'2020)
+- [x] [ActivityNet](/tools/data/activitynet/README.md) \[ [Homepage](http://activity-net.org/) \] (CVPR'2015)
```
Comment (dreamerlin) [inline README_cn.md:137]: Homepage -> 主页 ？

File: README_cn.md (hunk 2 above)
Comment (innerlee) [reply README_cn.md:174]: That's better

File: README_cn.md (hunk 3 above)
Comment (innerlee) [reply README_cn.md:137]: ok

**Review by innerlee — APPROVED** [review]

**Pull request merged**